app/main/controllers/template/rtmedia-functions.php

```diff
@@ -1559,13 +1559,16 @@ function rtmedia_pagination_next_link() {
 
 			// if there are more media than number of media per page to show than $rtmedia_query->media->media_id will be set other wise take media_id of very first media.
 			// For more understanding why array became object check rewind_media() in RTMediaQuery.php file and check it's call.
-			$post_id = ( isset( $rtmedia_query->media->media_id ) ? $rtmedia_query->media->media_id : $rtmedia_query->media[0]->media_id );
-			$post    = get_post( get_post_field( 'post_parent', $post_id ) );
+			if ( false !== $rtmedia_query->media ) {
```

Comment:
It is good to use !empty() check instead of checking strictly with a boolean. because current check will cause error if the check returns any falsy value which is not a bolean false, like empty string or null etc will cause issue.